lib/translator/inquiry.cc

```diff
@@ -236,6 +236,58 @@ scsi_defs::UnitSerialNumber BuildUnitSerialNumberVpd() {
   return TranslateUnitSerialNumberVpdResponse(identify_namespace_data);
 }
 
+scsi_defs::BlockLimitsVpd BuildBlockLimitsVpd() {
```

Comment:
Can this function be omitted and the caller just calls `TranslateBuildBlockLimitsVpdResponse` directly?